bower.json

```diff
@@ -12,7 +13,7 @@
     "angular-ui-utils": "bower",
     "angular-ui-router": "~0.2",
     "angular-file-upload": "1.1.5",
-    "angular-messages": "1.3.17",
+    "angular-messages": "~1.3",
```

Comment:
Can we leave the .17 so we know what version it was brought in and tested with? That's helpful when a new version is released and it contains some random bug and we need to revert.